README.rst

```diff
@@ -414,6 +413,31 @@ Example usage:
 
     client = ovh.Client()
 
+Use v1 and v2 API versions
+--------------------------
+
+When using OVHcloud APIs (not So you Start or Kimsufi ones), you are given the
+opportunity to aim for two API versions. For the European API, for example:
+
+- the v1 is reachable through https://eu.api.ovh.com/v1
+- the v2 is reachable through https://eu.api.ovh.com/v2
+- the legacy URL is https://eu.api.ovh.com/1.0
+
+ Calling ``client.get``, you can target the API version you want:
+
+.. code:: python
+
+    client = ovh.Client(endpoint="ovh-eu")
+
+    # Call to https://eu.api.ovh.com/v1/xdsl/xdsl-yourservice
+    client.get("/v1/xdsl/xdsl-yourservice")
```

Comment:
Seen together, this call raises an `ovh.exceptions.BadParametersError: Invalid signature` error.

Reply:
Seen together, linked to internal infrastructure, not to this PR.